appinfo/app.php

```diff
@@ -62,8 +62,9 @@ function () use ($c, $appName) {
 $request = $c->query('Request');
 if (isset($request->server['REQUEST_URI'])) {
 	$url = $request->server['REQUEST_URI'];
-	if (preg_match('%/apps/files(/.*)?%', $url)
-		|| preg_match('%^((?!/apps/).)*/s/\b(.*)\b(?<!/authenticate)$%', $url)
+	if (!preg_match('%/apps/files_drop/%', $url) &&
```

Comment:
if the gallery app needs "files" and "files_sharing", how about adding these two in an opt-in regexp instead of doing an opt-out of files_drop ?